.circleci/config.yml

```diff
@@ -77,7 +77,12 @@ jobs:
             git checkout master
             CURR_MBT_VERSION=$(cat ./VERSION)
             echo "Current release: ${CURR_MBT_VERSION}"
-            MBT_VERSION=$(echo $CURR_MBT_VERSION | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g')
+            if [[-z "$MBT_VERSION_MANUAL"]]; then 
+              echo "MBT_VERSION manual!!!"
```

Comment:
Please change to a message like `Using manual MBT version` with the version number

Reply:
Done